float32 lin_vel
float32 r_curv
float32 ang_vel
uint8 sign